needy/library.py

```diff
@@ -106,50 +106,58 @@ def build(self):
 
         self.__log_env_overrides(env_overrides)
         with OverrideEnvironment(env_overrides):
-            post_clean_commands = configuration['post-clean'] if 'post-clean' in configuration else []
-            with cd(self.project_root()):
-                for cmd in self.evaluate(post_clean_commands):
-                    command(cmd)
+            self.__clean()
```

Comment:
I don't think this is the right name. The cleaning happens above. I'd suggest naming it `__post_clean`.